docs/en/example/publiccode.yml

```diff
@@ -92,7 +92,7 @@ maintenance:
     - name: Francesco Rossi
       email: "francesco.rossi@comune.reggioemilia.it"
       affiliation: Comune di Reggio Emilia
-      phone: +39 231 13215112
+      phone: "+39 231 13215112"
```

Comment:
this is already a string as it contains spaces, and doesn't _need_ quotes.
I am a fan of making things more clear, so I am positive about this change, I am simply pointing out that it's not a real change of the specifictation.

Reply:
True, but I thought that this could be the extreme example: if we put double quotes also on a string which is already interpreted as a string in YAML we are passing the idea that it must be *explicit*.

Reply:
We observed several cases where phone numbers had different format, see below:

```
>>> yaml.safe_load('phone: 01234567')
{'phone': 342391}

>>> yaml.safe_load('phone: 023113215112')
{'phone': 2569869898}

>>> yaml.safe_load('phone: +39023598745')
{'phone': 39023598745}
```
That's why we need quoting in specification.

https://github.com/go-yaml/yaml/pull/86#issuecomment-144103070

[ref](https://yaml.org/type/int)

Reply:
I agree with both of you :-) 
We can't mandate the usage of quotes, because that depends on the rules of YAML and the heuristics of the libraries. But it is useful to use double quotes in our examples because that reminds to people who write publiccode.yml files by hand that it's better to use double quotes. Otherwise we should explain "this example does not use quotes because it contains spaces". :-)

Reply:
What @alranel wrote is precisely what I meant! Let's keep quotes for clarity but, technically, that field was already a string